service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsController.java

```diff
@@ -104,9 +116,13 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
     @PostMapping("/submitted")
     public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
         CaseData caseData = mapper.convertValue(callbackRequest.getCaseDetails().getData(), CaseData.class);
+
         if (isInGatekeepingState(callbackRequest.getCaseDetails())
             && standardDirectionsService.hasEmptyDates(caseData)) {
             applicationEventPublisher.publishEvent(new PopulateStandardDirectionsOrderDatesEvent(callbackRequest));
         }
+
+        applicationEventPublisher.publishEvent(new NewHearingsAddedEvent(callbackRequest));
```

Comment:
Being picky here (from the domain perspective)

If we name the NewHearingsAdded it should be ONLY raised if the new hearings are added.

I remember it was originally called HearingsUpdated which is different (but again we should detect if hearings were updated). It's tricky but the more correct you are with events the easier it's for consumers to consume.

Additionally, I would remove Event from the name, it seems for me as "smurph naming" 
NewHearingsAdded already denots the event (and the package name as well). Please consider refactoring.

Reply:
have renamed to NewHearingsAdded and surrounded with if statement